packages/screens/Wallet/libs/ledger.ts

```diff
@@ -0,0 +1,258 @@
+// @ts-nocheck
```

Comment:
Same here imagine that is complicated to fit types ? Maybe the purpose of a next PR ?

Reply:
It looks like WIP, and most of ts issues are from @cosmjs/* packages so hard to fix at the moment.